packages/website/cypress/step-definitions/base.ts

```diff
@@ -5,23 +5,19 @@ Given('User opens the {string} page', (path: string) => {
 });
 
 When('User clicks on the {string} link', (link: string) => {
-  cy.get(`a[href*="${link}"]`).click();
+  cy.get(`a[href*="${link}"]`).first().click();
 });
 
 When('User clicks on the {string} button', (button: string) => {
-  cy.get(`button:contains("${button}")`).click();
+  cy.contains('button',`${button}`).click();
 });
 
-When('User clicks on the button with id {string}', (id: string) => {
-  cy.get(`#${id}`).click();
+When('User clicks on the {string} tab', (link: string) => {
+  cy.contains('a', `${link}`).click();
 });
 
-When('User clicks on the button with {string} {string}', (property: string, label: string) => {
-  cy.get(`button[${property}="${label}"]`).click();
-});
-
-When('User clicks on the {string} element with text {string}', (element: string, text: string) => {
-  cy.get(element).contains(text).click();
+When('User clicks on the {int}st/nd/rd/th button or link with id {string}', (idx: number, id: string) => {
```

Comment:
I would change "button or link" for just "element", IMHO it generates more readable tests
```suggestion
When('User clicks on the {int}st/nd/rd/th element with id {string}', (idx: number, id: string) => {
```

Reply:
Fixed it as your suggestion!

![Screenshot 2025-02-24 at 10 47 58](https://github.com/user-attachments/assets/d24a7a6a-5b8c-4bc2-95ef-5c018a8ae2dc)